HW_2_updated.py

```diff
@@ -0,0 +1,45 @@
+import random
+import string
+
+def get_the_list():
+    uni_list = []
+    # get the number of key/values in a dict + dicts quantity
+    quantity = random.randint(2,10)
```

Comment:
it'd be better to move "quantity" and "range number variables (0, 100)" to function definition in order to make function more reusable.